doc/code/Apps.md

```diff
@@ -35,18 +35,20 @@ that will call the method `Refresh()` periodically.
 
 ## App types
 
-There are basically 2 types of applications : **system** apps and **user** apps.
+There are basically 3 types of applications : **system** apps and **user** apps and **watch faces**.
 
 **System** applications are always built into InfiniTime, and InfiniTime cannot work properly without those apps.
-The watchfaces, settings, notifications and the application launcher are examples of such system applications.
+The watch faces, settings, notifications and the application launcher are examples of such system applications.
```

Comment:
As of this PR, watch faces aren't system applications.

Reply:
Good catch! I removed watch faces from the list!